packages/extras/src/lib/components/Align/Align.svelte

```diff
@@ -124,10 +124,7 @@
 >
   <T is={outerGroup}>
     <T is={innerGroup}>
-      <slot
-        align={scheduleAligning}
-        ref={group}
-      />
+			{@render children(group, scheduleAligning)}
```

Comment:
Based on our discussions, it looks like this is the pattern we'd want to adopt:

```suggestion
			{@render children({ align: scheduleAligning, ref: group })}
```